src/core/namespace.c

```diff
@@ -3967,6 +3970,201 @@ static const char* const private_bpf_table[_PRIVATE_BPF_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(private_bpf, PrivateBPF);
 
+#include "bpf-delegate-configs.inc"
+
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_cmd, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_map_type, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_prog_type, uint64_t);
+DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_attach_type, uint64_t);
+
+char* bpf_delegate_commands_to_string(uint64_t u) {
+        if (u == UINT64_MAX)
+                return strdup("any");
+
+        _cleanup_free_ char *buf = NULL;
+
+        BIT_FOREACH(i, u)
+                if (strextendf_with_separator(&buf, ",", "%s", bpf_delegate_cmd_to_string(i)) < 0)
+                        return NULL;
+
+        return TAKE_PTR(buf);
+}
+
+int bpf_delegate_commands_from_string(const char *s, uint64_t *ret) {
```

Comment:
Also this one. I'd like to make this take a function pointer, then we can merge the for parsers.